src/render/utils/animation.ts

```diff
@@ -167,6 +169,26 @@ function animateTarget(
             ...transition,
         })
 
+        /**
+         * Layout animations rely on synchronous rendering of layout, which
+         * is incompatible with animating layout-affecting properties.
+         * Here we check if this component or any above it are animating layout
+         * and warn if an animation is started on some popular layout-effecting properties.
```

Comment:
Only the popular ones, or all of them?

Reply:
Just the popular ones, I don't think we should add paddingLeft etc unless this comes up